content/docs/faq-ajax.md

```diff
@@ -65,9 +65,9 @@ class MyComponent extends React.Component {
   render() {
     const { error, isLoaded, items } = this.state;
     if (error) {
-      return <div>Error: {error.message}</div>;
+      return <div>Սխալ\` {error.message}</div>;
     } else if (!isLoaded) {
-      return <div>Loading...</div>;
+      return <div>Բեռնվում է...</div>;
```

Comment:
Loading... -» Բեռնում...

Is Loading... -» Բեռնվում է...

Քոնը առաջի դեպքն ա